app/src/main/res/layout/fragment_categorization.xml

```diff
@@ -25,21 +25,23 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:hint="@string/categories_search_text_hint"
-            android:imeOptions="flagNoExtractUi"
+            android:maxLines="1"
+            android:gravity="left"
             android:inputType="textCapWords"
-            android:maxLines="1" />
+            android:imeOptions="flagNoExtractUi"/>
 
         <ProgressBar
             android:id="@+id/categoriesSearchInProgress"
-            style="?android:progressBarStyleSmall"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_gravity="center_vertical|end"
-            android:layout_marginEnd="@dimen/tiny_gap"
-            android:layout_marginRight="@dimen/tiny_gap"
             android:indeterminate="true"
             android:indeterminateOnly="true"
-            android:visibility="gone" />
+            android:layout_marginRight="@dimen/tiny_gap"
+            android:layout_marginEnd="@dimen/tiny_gap"
+            android:layout_gravity="center_vertical|right"
```

Comment:
It seems like the only change is this line. But places of other lines are changed. I couldn't understand the reason of this changes.